lms/static/sass/course/instructor/_instructor.scss

```diff
@@ -1,189 +0,0 @@
-.instructor-dashboard-wrapper {
```

Comment:
Yeah, looks like we rely on ``.instructor-dashboard-wrapper-2`` for the current instructor dashboard UI rendering. Burn this motha-selecta down!